ui/send_page.go

```diff
@@ -222,72 +219,16 @@ func (pg *SendPage) Handle(common pageCommon) {
 	}
 }
 
-func (pg *SendPage) drawWalletsTab(common pageCommon, body func()) {
-	wallets := make([]decredmaterial.TabItem, len(pg.wallets))
-	for i := range pg.wallets {
-		wallets[i] = decredmaterial.TabItem{
-			Label: pg.theme.Body1(pg.wallets[i].Name),
-		}
-	}
-	pg.walletsTab.SetTabs(wallets)
-
-	pg.setAccountTabs()
-	pg.walletsTab.Layout(common.gtx, func() {
-		layout.Flex{Axis: layout.Horizontal}.Layout(common.gtx,
-			layout.Rigid(func() {
-				layout.Inset{Top: unit.Dp(10), Right: unit.Dp(10)}.Layout(common.gtx, func() {
-					pg.theme.H6("Accounts: ").Layout(common.gtx)
-				})
-			}),
-			layout.Rigid(func() {
-				pg.accountsTab.Layout(common.gtx, body)
-			}),
-		)
-	})
-}
-
-func (pg *SendPage) setAccountTabs() {
-	accounts := make([]decredmaterial.TabItem, len(pg.selectedWallet.Accounts))
-	for i := range pg.selectedWallet.Accounts {
-		if pg.selectedWallet.Accounts[i].Name == "imported" {
-			continue
-		}
-		accounts[i] = decredmaterial.TabItem{
-			Label: pg.theme.Body1(pg.selectedWallet.Accounts[i].Name),
-		}
-	}
-	pg.accountsTab.SetTabs(accounts)
-}
-
 func (pg *SendPage) Layout(common pageCommon) {
 	if len(common.info.Wallets) == 0 {
-		// show no wallets message
+		common.Layout(common.gtx, func() {
+			layout.Center.Layout(common.gtx, func() {
+				common.theme.H3("No wallet loaded").Layout(common.gtx)
```

Comment:
When there's no wallet loaded, the app automatically displays the "create wallet" page. Or is there a behaviour that needs this?